src/Altinn.Platform/Altinn.Platform.Events/Events/Altinn.Platform.Events.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.6.1" />
     <!-- Major version update (3.1 -> 5.0) to be done in separate issue: #5587 -->
```

Comment:
seems the major update has been done so the comment can be removed

Reply:
Yeah, was fixed by @TheTechArch during the weekly update.